src/desktop/lib/global_client_setup.tsx

```diff
@@ -4,10 +4,10 @@
  * @see https://github.com/artsy/force/blob/master/src/lib/middleware/assetMiddleware.ts
  */
 if (process.env.NODE_ENV === "production") {
-  __webpack_public_path__ = "https://d1rmpw1xlv9rxa.cloudfront.net/assets/"
-
-  // @ts-ignore
-  window._logAssetPath = () => process.env.CDN_URL + "/assets/"
+  __webpack_public_path__ =
+    (window.location.hostname === "www.artsy.net"
+      ? process.env.CDN_PRODUCTION_URL
+      : process.env.CDN_STAGING_URL) + "/assets/"
```

Comment:
If we're on prod we'll use the production url, otherwise we'll use staging. This isn't necessarily ideal, but it'll do for now.